src/main/java/com/heycar/pairingsession/controller/ListingsController.java

```diff
@@ -0,0 +1,56 @@
+package com.heycar.pairingsession.controller;
+
+import com.heycar.pairingsession.service.ListingsService;
+
+public class ListingsController {
+
+    private final ListingsService listingsService;
+
+    public ListingsController(ListingsService listingsService) {
+        this.listingsService = listingsService;
+    }
+
+    /** Should accept the following request:
+     curl -X POST 'http://localhost:8080/listings/{dealerId}/' \
```

Comment:
Do we want to give the url to them, for me its interesting to see if they come up with one that has good REST principles

Reply:
hmm yea makes sense! will update it